packaging/rpm/cvmfs-universal.spec

```diff
@@ -31,6 +31,11 @@
   %define build_snapshotter 1
 %endif
 
+# flag to control building of unittests.
+# No tests are built by default, can be overriden 
+# with rpmbuild --define="build_testing 1"
+%{!?build_testing: %define build_testing 0}
```

Comment:
Defining this by default to 0 isn't really necessary since the ifs already allow for it to be undefined, but I guess it's good for documentation.